File: java/Car.java
public class Car {
    Integer id;
    String license;
    Driver driver;
    private Integer passegenger;

    public Car(String license, Driver driver) {
        this.license = license;
        this.driver = driver;
    }

    void printData() {
        System.out.print("Car License: " + license + ", Driver : " + driver.name);
        System.out.println(passegenger != null ? ", Passegenger: " + passegenger : "");
    }
    public Integer getPassenger(){
        return passegenger;
    }
    public void setPassenger(Integer passegenger){
        if(passegenger == 4){
            this.passegenger = passegenger;
        } else {
            System.out.println("El numero de passegenger: debe ser igula a cuatro");
        }
    }
}

File: js/Car.js
class Car {
    constructor(license, driver) {
        this.id = null;
        this.license = license;
        this.driver = driver;
        this.passengenger = null;
    }
    
    printData() {
        console.log(`Card license: ${this.license}`);
        console.log('Card driver: ' + this.driver.name);
    }
    get getPassenger(){
        return this.passengenger;
    }
    set setPassenger(pasenger) {
        if(pasenger === 4){
            this.passengenger = pasenger;
        } else {
            console.error("passengenger dese ser: 4")
        }
    }
}


export default Car;